src/Sylius/Bundle/CurrencyBundle/Templating/Helper/CurrencyHelper.php

```diff
@@ -53,12 +59,14 @@ public function __construct(
         CurrencyContextInterface $currencyContext,
         CurrencyConverterInterface $converter,
         MoneyFormatterInterface $moneyFormatter,
-        CurrencyProviderInterface $currencyProvider
+        CurrencyProviderInterface $currencyProvider,
+        LocaleContext $localeContext
```

Comment:
You cannot inject _LocaleContext_ here, because _CurrencyBundle_ doesn't know anything about locale component.